packages/screens/Settings/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface SettingItemGroupType {
+  [key: string]: SettingItemType;
+}
+
+export interface SettingItemType {
+  title: string;
+  description: string;
+  state: string | boolean;
```

Comment:
can we avoid to mix string | bool ? is that possible to convert value before set it to `state` to keep a consistant type